package.json

```diff
@@ -12,7 +12,8 @@
   "main": "./dist.js",
   "scripts": {
     "prepublish": "./node_modules/.bin/webpack --bail",
-    "start": "make serve",
+    "start": "webpack-dev-server",
+    "start": "webpack --colors --progress",
```

Comment:
Like @dekrain said, this doesn't add the script `build`, it adds a second `start` script which makes this invalid JSON. Please update to `build`.